### PR TITLE
jetbrains_mono, new font, based on the work of 0xtrzy

### PR DESCRIPTION
**Pull request by Begasus**
Co-authored with @0xtrzy

### CODE REVIEW

**Comment by Begasus** [comment]
Original recipe: https://gitlab.com/0xtrzy/jetbrainsmono-package

File: media-fonts/jetbrains_mono/jetbrains_mono-1.0.3.recipe [hunk 1]
```diff
@@ -0,0 +1,37 @@
+SUMMARY="A monospace font designed for programmers"
+DESCRIPTION="JetBrains Mono’s typeface forms are simple and free \
+from unnecessary details. Rendered in small sizes, the text looks crisper.
+
+The easier the forms, the faster the eye perceives them \
+and the less effort the brain needs to process them."
+HOMEPAGE="https://www.jetbrains.com/lp/mono/"
+COPYRIGHT="2020 Philipp Nurullin \
+	2020 Konstantin Bulenkov \
+	The whole JetBrains Team"
```
Comment (Begasus) [inline media-fonts/jetbrains_mono/jetbrains_mono-1.0.3.recipe]: Not completely sure here (COPYRIGHT):
https://github.com/JetBrains/JetBrainsMono#credits

Comment (AlwaysLivid) [reply media-fonts/jetbrains_mono/jetbrains_mono-1.0.3.recipe]: The repository does contain code from the two authors, which is licensed under the Apache license.

I'll open an issue on their repository just to make sure.

e.g. https://github.com/JetBrains/JetBrainsMono/blob/master/scripts/UpdateMetaData.kt

Comment (AlwaysLivid) [reply media-fonts/jetbrains_mono/jetbrains_mono-1.0.3.recipe]: Proposed a relevant change to your repository, @Begasus.

Comment (AlwaysLivid) [reply media-fonts/jetbrains_mono/jetbrains_mono-1.0.3.recipe]: Also opened this issue, just to be sure: https://github.com/JetBrains/JetBrainsMono/issues/138

Comment (Begasus) [reply media-fonts/jetbrains_mono/jetbrains_mono-1.0.3.recipe]: Thanks :)

Comment (bulenkov) [reply media-fonts/jetbrains_mono/jetbrains_mono-1.0.3.recipe]: That's fine

**Pull request merged**